modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -804,17 +875,18 @@ public void testSendNotificationWithRoles() throws Exception {
 					RandomTestUtil.randomString(),
 					NotificationConstants.TYPE_EMAIL, Collections.emptyList()));
 
-		_accountEntryUserRelLocalService.addAccountEntryUserRels(
-			accountEntry1.getAccountEntryId(),
-			new long[] {user1.getUserId(), user2.getUserId()});
-		_accountEntryUserRelLocalService.addAccountEntryUserRels(
-			accountEntry2.getAccountEntryId(), new long[] {user3.getUserId()});
-		_organizationLocalService.addUserOrganization(
-			user4.getUserId(), organization1.getOrganizationId());
-		_organizationLocalService.addUserOrganization(
-			user5.getUserId(), organization2.getOrganizationId());
-		_organizationLocalService.addUserOrganization(
-			user6.getUserId(), childOrganization.getOrganizationId());
+		_objectActionLocalService.updateObjectAction(
```

Comment:
Isolate this piece of code into a method

Reply:
Done.